frontend/src/hooks/api/getters/useMessageBanners/useMessageBanners.ts

```diff
@@ -0,0 +1,38 @@
+import { useMemo } from 'react';
+import { formatApiPath } from 'utils/formatPath';
+import handleErrorResponses from '../httpErrorResponseHandler';
+import { useConditionalSWR } from '../useConditionalSWR/useConditionalSWR';
+import useUiConfig from '../useUiConfig/useUiConfig';
+import { useUiFlag } from 'hooks/useUiFlag';
+import { IInternalMessageBanner } from 'interfaces/messageBanner';
+
+const ENDPOINT = 'api/admin/message-banners';
```

Comment:
This is re-defined here, I believe you can just export this const in the API definition